silver/tests/spec/test_documents.py

```diff
@@ -41,9 +41,11 @@ def setUp(self):
 
     def _get_expected_data(self, document, transactions=None):
         kind = unicode(document.kind.lower())
+        payment_method_url = u'http://testserver/customers/%d/payment_methods/%d/'
```

Comment:
Doing `http://testserver + reverse(view, kwargs)` would be nice for all these urls, but it's not necessarily related to this PR.